_posts/2020-07-20-notes-from-community-call-2-july-2020.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Notes from community call - 2 July 2020"
+date: 2020-07-20
+author: Jan Ainali
+type: blogpost
+excerpt: Splitting 'Welcome contributions' in the Standard for Public Code
+categories:
+  - Community call
+---
+
+# 2 July 2020 Standard for Public Code community call
+
+## Update from the Foundation
+
+* The first Standard compliance analysis [has been merged](https://github.com/open-zaak/open-zaak/pull/609)!
+* There's [a mailing list for the Standard for Public Code](https://lists.publiccode.net/mailman/postorius/lists/standard.lists.publiccode.net/).
+
+## Attendees
+
+* [Jan Ainali](https://publiccode.net/team/jan-ainali.html) (chair)
+* [Eric Herman](https://publiccode.net/team/eric-herman.html)
+* [Boris van Hoytema](https://publiccode.net/team/boris-van-hoytema.html)
+* [Alba Roza](https://publiccode.net/team/alba-roza.html)
+* [Elena Findley-de Regt](https://publiccode.net/team/elena-findley-de-regt.html)
+* [Laura Scheske](https://publiccode.net/team/laura-scheske.html)
+* [Jonathan Pichot](https://twitter.com/_pichot)
+* [Peter Kuhn](https://twitter.com/nhuKreteP)
+
+## Notes
+
+### Introductions
+
+Jonathan and Peter was given an introduction to the Foundation for Public Code and an explanation of codebase stewardship. Rather than attempt to go into details of the Standard for Public Code, they each decided to come back to a Foundation for Public Code community call in the future.
```

Comment:
```suggestion
Jonathan and Peter were given an introduction to the Foundation for Public Code and an explanation of codebase stewardship. Rather than attempt to go into details of the Standard for Public Code, they each decided to come back to a Foundation for Public Code community call in the future.
```